faraday/proxy.py

```diff
@@ -581,7 +581,12 @@ def main():
 
     # Associate serial ports with callsigns
     # global units
-    units = callsign2COM()
+    try:
+        units = callsign2COM()
+
+    except ConfigParser.NoSectionError as e:
+        logging.error(e)
+        sys.exit()
```

Comment:
On error Python should exit with a non-zero error code so when catching errors like this we should use `sys.exit(1)`. (Coming up with a scheme where 1 means configuration error, 2 means something else, etc. is entirely optional and more work than really necessary.)